src/desktop/src/index.js

```diff
@@ -79,6 +79,18 @@ new Promise((resolve, reject) => {
         // Assign accountIndex to every account in accountInfo if it is not assigned already
         store.dispatch(assignAccountIndexIfNecessary(get(data, 'accounts.accountInfo')));
 
+        if (Electron.mode === 'tray') {
+            // Add Realm change listener to sync read-only Tray app with Main app
+            realm.addListener('change', () => {
```

Comment:
Nice!

Reply:
It's removed when the app quits and should be always active while the app runs.